nemo_skills/inference/model/tool_call.py

```
@@ -59,22 +58,26 @@ def __init__(
+            tool_name = tool_call["name"]
+            tool_args = tool_call["arguments"]
+        else:
+            raise ValueError(f"Unsupported completion type for tool call: {completion_type}")
 
         ##
         # TODO(sanyamk): Not all tool arguments might necessarily be in JSON format.
         #   Kept here to handle errors for now.
         try:
             tool_args = json.loads(tool_args)
         except json.decoder.JSONDecodeError as e:
+            LOG.error(f"Tool arguments are not in JSON format: {tool_args}")
```

Ruff (0.13.2)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)